src/solver/conv_direct_naive_conv.cpp

```diff
@@ -332,6 +332,784 @@ void conv_internal::DebugPrintTensorStrides(const TensorDescriptor& inDesc,
     printOneStrideVec("outDesc = ", outDesc.GetStrides());
 }
 
+namespace conv_internal {
+::miopen::solver::ConvSolution
+GetConv2DFWDSolution(const ExecutionContext& ctx, const ::miopen::conv::ProblemDescription& problem)
+{
+    ::miopen::solver::ConvSolution result;
+
+    int hi          = problem.GetInHeight();
```

Comment:
We need to use `ProblemInterpreter` here because it corrects for the case when descriptors (and buffers) are swapped in the backward passes.